src/components/ListBooks.js

```diff
@@ -0,0 +1,42 @@
+const ListBooks = (props) => {
```

Comment:
- [ ] You are requested to have a book component and a booklist component, in the booklist you will map through all the books that you may receive and create a book component for each one, right now, you are doing both things in the same place, please divide this component in order to comply with the project requirements.